sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -853,14 +853,18 @@ DROP TABLE `eligibility_response`;
 ALTER TABLE `x12_partners` CHANGE `processing_format` `processing_format` ENUM('standard','medi-cal','cms','proxymed','oa-eligibility','avality-eligibility') DEFAULT NULL;
 #EndIf
 
-#IfMissingColumn eligibility_id insurance_companies
+#IfMissingColumn insurance_companies eligibility_id
 ALTER TABLE `insurance_companies` ADD `eligibility_id` VARCHAR(32) DEFAULT NULL;
 #EndIf
 
-#IfMissingColumn x12_default_eligibility_id insurance_companies
+#IfMissingColumn insurance_companies x12_default_eligibility_id
 ALTER TABLE `insurance_companies` ADD `x12_default_eligibility_id` INT(11)  DEFAULT NULL;
 #EndIf
 
 #IfMissingColumn users_secure login_fail_counter
 ALTER TABLE `users_secure` ADD `login_fail_counter` INT(11) DEFAULT '0';
 #EndIf
+
+#IfMissingColumn x12_partners x12_dtp03
+ALTER TABLE `x12_partners` ADD `x12_dtp03` CHAR(1) DEFAULT 'A';
```

Comment:
Also need to add change to database.sql

Reply:
don't know what's wrong with me. very forgetful of late! resolved

Reply:
lol, resolved the issue that is. forgetfulness still an issue though!